_posts/2024-10-23-arrow-go-18.0.0-release.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "Apache Arrow Go 18.0.0 Release"
+date: "2024-10-23 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the v18.0.0 release of
+Apache Arrow Go. This release covers 35 resolved issues from
+11 contributors.
+
+## Release Highlights
+
+### Move to new Repository
+
+The Apache Arrow Go implementation has been shifted out of the Arrow monorepo and to its
+own repository now located at https://github.com/apache/arrow-go. This will enable fewer
+major version releases and facilitate more minor version/patch releases instead as it will
+detach the version number from the Arrow C++ and other implementations in the monorepo.
+Current users of Arrow Go will want to carefully note the breaking change to module paths this move causes. To upgrade to Arrow Go v18.0.0, ensure you properly update all relevant module paths in your `go.mod`, replacing all instances of `github.com/apache/arrow/go/` with `github.com/apache/arrow-go`.
+Existing Go related issues have been moved to the new repository and please file any new
+issues on the new repository instead of the Arrow monorepo.
```

Comment:
```suggestion
The Apache Arrow Go implementation has been shifted out of the Arrow monorepo
and to its own repository now located at https://github.com/apache/arrow-go.
This will enable fewer major version releases and facilitate more minor
version/patch releases instead as it will detach the version number from the
Arrow C++ and other implementations in the monorepo.

Current users of Arrow Go will want to carefully note the breaking change to
module paths this move causes. To upgrade to Arrow Go v18.0.0, ensure you
properly update all relevant module paths in your `go.mod`, replacing all
instances of `github.com/apache/arrow/go/` with `github.com/apache/arrow-go`.

Existing Go related issues have been moved to the new repository and please file
any new issues on the new repository instead of the Arrow monorepo.
```

My previous suggestion borked the formatting here. The above suggestion should fix it. Sorry about that!